tests/lib/dune

```diff
@@ -0,0 +1,10 @@
+(test
+  (name      patcher)
+  (modules   patcher)
+  (libraries opam-core))
+
+(test
```

Comment:
Should we add in CI at least a run of it ? Maybe not in all jobs

Reply:
I don't follow - this test is run in CI (as part of `make tests`) isn't it?

Reply:
True, I had in mind that it is `make reftests` that is run in CI.